spanner/spring-data/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="https://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+
+  <!--
+    The parent pom defines common style checks and testing strategies for our samples.
+    Removing or replacing it should not affect the execution of the samples in anyway.
+  -->
+  <parent>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.0.11</version>
+  </parent>
+
+  <groupId>com.example.spanner</groupId>
+  <artifactId>spanner-google-cloud-spring-data-sample</artifactId>
+  <name>Spring Data Spanner Code Sample</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework.cloud</groupId>
+      <artifactId>spring-cloud-gcp-starter-data-spanner</artifactId>
+      <version>1.1.1.RELEASE</version>
```

Comment:
Can we instead demonstrate the use of the Spring Cloud BOM here instead, which is what we recommend?
I know it's not critical for this small sample, but if the users adopt this sample for a larger application that brings in other Spring components, the BOM would be super helpful.

Reply:
Hmmm should I hold off on using that Greenwich.SR1 BOM ? cuz you know...

Reply:
Hmm perhaps. In that case, let's use our own BOM `spring-cloud-gcp-dependencies'.

Reply:
added bom